docs/lakehouse-iceberg-catalog.md

```diff
@@ -264,24 +264,24 @@ You can pass [Iceberg table properties](https://iceberg.apache.org/docs/1.5.2/co
-| `creator`                       | The table creator.                                      |
-| `current-snapshot-id`           | The snapshot represents the current state of the table. |
-| `cherry-pick-snapshot-id`       | Selecting a specific snapshot in a merge operation.     |
-| `sort-order`                    | Selecting a specific snapshot in a merge operation.     |
-| `identifier-fields`             | The identifier fields for defining the table.           |
-| `write.distribution-mode`       | Defines distribution of write data                      |
+| Configuration item        | Description                                             | Default value | Required | Since Version |
+|---------------------------|---------------------------------------------------------|---------------|----------|---------------|
+| `comment`                 | The table comment.                                      | None          | No       | 0.2.0         |
+| `creator`                 | The table creator.                                      | None          | No       | 0.2.0         |
+| `current-snapshot-id`     | The snapshot represents the current state of the table. | None          | No       | 0.2.0         |
+| `cherry-pick-snapshot-id` | Selecting a specific snapshot in a merge operation.     | None          | No       | 0.2.0         |
+| `sort-order`              | Selecting a specific snapshot in a merge operation.     | None          | No       | 0.2.0         |
+| `identifier-fields`       | The identifier fields for defining the table.           | None          | No       | 0.2.0         |
+| `write.distribution-mode` | Defines distribution of write data                      | None          | No       | 0.2.0         |
 
 Gravitino server doesn't allow to change such properties:
 
-| Configuration item | Description                                  |
-|--------------------|----------------------------------------------|
-| `location`         | Iceberg location for table storage.          |
-| `format`           | Iceberg table format like `iceberg/parquet`. |
-| `format-version`   | Iceberg meta data format version.            |
-| `provider`         | Table provider like `iceberg`.               |
+| Configuration item | Description                                  | Default value | Required | Since Version |
+|--------------------|----------------------------------------------|---------------|----------|---------------|
+| `location`         | Iceberg location for table storage.          | None          | No       | 0.2.0         |
+| `format`           | Iceberg table format like `iceberg/parquet`. | None          | No       | 0.2.0         |
+| `format-version`   | Iceberg meta data format version.            | None          | No       | 0.2.0         |
+| `provider`         | Table provider like `iceberg`.               | None          | No       | 0.2.0         |
```

Comment:
Since users can pass all the properties provided by Iceberg officials, what is the difference between these properties and official properties? Does the description seem too simple?

Reply:
it's immutable and some properties are not provided by iceberg like `provider`

Reply:
@jerqi Can you help review these changes?